docs/project-app.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Create Flow app from scaffold
+sidebar_title: Create App
+description: How to create a Flow app with recommended structure
+---
+
+The Flow CLI provides a command to create an app from examples
+and templates. 
+
+```shell
+flow app create
+```
+
+## Example Usage
+
+```shell
+> flow app create my-app
+
+Enumerating objects: 349, done.
+Counting objects: 100% (349/349), done.
+Compressing objects: 100% (261/261), done.
+Total 349 (delta 55), reused 347 (delta 53), pack-reused 0
+? Which do you want to start with? Template
```

Comment:
The question I mean is if a user wants to start from an (example) app that can be built and run (kitty-items) with changing very few or no code or a template have only necessary files (framework, scaffold). Since I'm not a native English speaker, could you recommend a clear and succinct question, please?

Reply:
Hm.. maybe you can set it up like this? (idk, maybe look at create-react-app for inspiration!)
```
Project scaffolding
> Start with sample app
   Minimal
```